README.md

```diff
@@ -137,7 +137,7 @@ We're definitely not doing that. With react_on_rails, webpack is mainly generati
   foreman start -f Procfile.dev
   ```
 
-7. Visit [localhost:3000/hello_world](http://localhost:3000/hello_world)
+7. Visit [localhost:3000/hello_world](http://localhost:3000/hello_world) or [localhost:5000/hello_world](http://localhost:5000/hello_world) if the former doesn't work.
```

Comment:
Just looke at my Procfile and it seems to be missing the '-p 3000'. I'll add that in. Any ideas why it might not have generated that properly for me? @jbhatab